libraries/Microsoft.Bot.Schema/CardEx.cs

```diff
@@ -4,9 +4,9 @@
 namespace Microsoft.Bot.Schema
 {
     /// <summary>
-    /// Extension methods for converting strongly typed Card objects to <see cref="Attachment"/>.
+    /// Extension methods for Cards.
     /// </summary>
-    public static partial class Extensions
+    public static class CardEx
```

Comment:
Shouldn't these methods be included in each class, Hero, Thumbnail, etc?, the ToAttachment?
I see it only does this.
```csharp
return new Attachment
{
    Content = card,
    ContentType = contentType,
};
```

Reply:
I considered adding the method to each class but it seemed a lot of duplicated code that could just be in one file to be reused.

Reply:
Right, maybe if we have a Card class where it only has related code to Cards, eg. the ToAttachment method, each specific card class could inherit it from the Card's one. But i dont know, its my opinion